tests/test_sticker.py

```diff
@@ -118,6 +120,7 @@ def test_expected_values(self, sticker):
         assert sticker.thumb.width == self.thumb_width
         assert sticker.thumb.height == self.thumb_height
         assert sticker.thumb.file_size == self.thumb_file_size
+        # assert sticker.premium_animation == self.premium_animation
```

Comment:
```suggestion
        # we need to be a premium TG user to send a premium sticker, so the below is not tested
        # assert sticker.premium_animation == self.premium_animation
```

Reply:
ahaha I added this earlier before finishing developing of the file and forgot about it. I like that its there